h/streamer/__init__.py

```diff
@@ -1,4 +1,6 @@
-# -*- coding: utf-8 -*-
```

Comment:
Deleting these coding comments as I go. One day I'll write a PR to get them all.

Reply:
Sounds like a job for REGEXES! No actually, I bet you can get them all (or most) with global find replace